tbx/project_styleguide/templates/patterns/pages/team/team_listing.html

```diff
@@ -2,13 +2,19 @@
 {% load wagtailcore_tags wagtailimages_tags static %}
 
 {% block content %}
+    <div class="grid">
+        {% include "patterns/molecules/title-filters/title-filters--team.html" with tags=tags item=page strapline=page.strapline %}
 
-    {% include "patterns/molecules/title-filters/title-filters.html" with tags=tags item=page %}
-
-    {{ page.strapline }}
-
-    {% for member in people %}
-        {% include "patterns/molecules/team-member/team-member.html" with item=member %}
-    {% endfor %}
-
+        <ul class="team-listing grid__team-listing">
+            {% for member in people %}
+                <li class="team-listing__item">
+                    {% srcset_image member.image format-webp fill-{230x230,370x374} sizes="(max-width: 598px) 370px, (min-width: 599px) 230px" class="team-listing__image" alt="" %}
```

Comment:
I've kept to these exact sizes for the images, to keep page load down, except at tablet where it looked a lot better if we had 3 across